models/public/hybrid-cs-model-mri/accuracy-check.yml

```diff
@@ -0,0 +1,18 @@
+models:
+  - name: wnet_20
+
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: image_processing
+          std: 1
+          cast_to_uint8: False
+
+    datasets:
+      - name: CalgaryCampinasSingleChannel_20perc_downsample
```

Comment:
please provide dataset definitions into <omz>/data/dataset_definitions.yml
and create soft link on config to <omz>/tools/accuracy_checker/configs